### ISSUE TITLE
a common problem with the mod and its concepts

### ISSUE THREAD

**Issue by TheVirusNVGM** [Kikugie/techutils#8]
I don’t understand the essence of the mod, I’m already trying to look at ready-made schematics - but the window where the isometric image should be is empty, I just don’t understand how to use this mod and what it is, there are no guides, no reviews, and the potential on the page is painted to the maximum. I would like to understand how to synchronize the world edit with an axе and a stick, how to view isometry, how the mod works in general, since information is freely available - 0

**Comment by Kikugie** [comment]
Mod's settings can be accessed from ModMenu
- `renderPreview` enables schematic 3D model, note however that it only works for .litematic files
- `autoWeSync` in WorldEdit tab enables the sync. It will sync to existing selection automatically and update when you change it.  

Thanks for your report, I see how it can be confusing and I'm going to update descriptions in the next release. Let me know if some feature is unclear.

**Comment by TheVirusNVGM** [comment]
Yes, there is a question - I have a quilt 1.19.2, I do not know if this is the problem, but still this is what my rendering looks like - 
![image](https://user-images.githubusercontent.com/73340173/236703053-3631e32f-f8d8-4b5d-b8c2-42dfb7c47ea7.png)
I do not really understand what the problem is, I think the format is lightmatic, but it is not visible. And a question off-topic, is it possible to change the blocks in the scheme itself? Let's say there is already a ready-made scheme, and I want to replace the cobblestone on the dirt, is it possible? If not, is it in your power to create such a function with your mod?

**Comment by Kikugie** [comment]
Ah, schematic previews are only available since 1.19.3 and generally new features are kept up to date with latest mc versions.  
Answering the other question: Litematica can do that, check it's [wiki page](https://github.com/maruohon/litematica/wiki/Schematic-Editing#litematicas-edit-schematic-mode-for-replacing-adding-or-removing-schematic-blocks).

**Comment by TheVirusNVGM** [comment]
will there be isometrics on 1.19.2?

**Comment by Kikugie** [comment]
There's quite a big change between 1.19.2 and 1.19.3 which breaks the render, so that is unlikely.

**Comment by TheVirusNVGM** [comment]
Great, thanks a lot for the info

**Comment by Am1noas** [comment]
try Schemetic Preview mod and disable preview feature in this mod (otherwise overlapping). that mod can also replace block while keeping nbt. Howevery idk if it is useable for '.lIgHtmATic' format

**Comment by kikugie** [comment]
Newer versions don't even have the schematic preview
